.github/workflows/centos7night/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM centos:7.7.1908
```

Comment:
Could you please argue why running nightly build for CentOS-7 make sense? It was released more then six years ago so it seems unlikely to me that suddenly some breaking changes will appear...

Reply:
The idea is to test binder and not CentOS7. Concerning the later: it is the production system and will be supported for the next 5-10 years. In our field 90%+ cluster machines run it.   Of course, you are right,  there are no incompatible changes and most of the things get only the most crucial updates, but for instance, cmake is updated on a regular basis. I have 3.17.1 now.

Reply:
Oh, i am not arguing of usability of CentOS-7, - it the same here, - all major clusters is using it now and will use it for quite a while. My point was that it is very stable environment so nightly tests seems a bit excessive. But i see your point of this being Binder "nightly" tests, (not the CentOS's one)